dev/devicelab/lib/framework/utils.dart

```diff
@@ -314,6 +314,13 @@ Future<int> exec(
       .transform<String>(const LineSplitter())
       .listen((String line) {
         print('stderr: $line');
+        // grab gradle daemon log file.
+        if (line.contains('log file:')) {
+          final File file = File(line.split(':').last.trim());
```

Comment:
If gradle crashes it writes a log file, this ensures that it shows up in our crash logs

Reply:
good call